packages/fluentui/docs/src/examples/components/Alert/Types/AlertExampleDismissible.shorthand.tsx

```diff
@@ -10,6 +10,7 @@ const AlertExampleDismissible = () => {
       content="This is a special notification which you can dismiss if you're bored with it."
       dismissible
       onVisibleChange={() => setVisible(false)}
+      dismissAction={{ 'aria-label': 'close' }}
```

Comment:
Please could you also add it to best practices block?

Reply:
added :)